http/README.md

```diff
@@ -19,6 +19,14 @@ At the same time, nodes that are already connected via a libp2p connection, will
 
 Any protocol that follows request-response semantics can easily be mapped onto HTTP (mapping protocols that don’t follow a request-response flow can be more challenging). Protocols are encouraged to follow best practices for building REST APIs. Once a mapping has been defined, a single implementation can be used to serve both traditional libp2p as well as libp2p-HTTP clients.
 
+Specifically, using libp2p+HTTP will allow:
+
+1. HTTP servers to offer same endpoints over HTTP and libp2p streams, allowing clients to reuse existing libp2p connections
```

Comment:
Is this a main selling point? I don't care if I have 2 connections vs a single one. Although I guess the reduced latency could be good.

Reply:
Definitely not. I'll move it down the list.

Reply:
Actually, I'll just remove it.